src/Simulation/TargetDefinitions/Decompositions/RxxFromExp.qs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Intrinsic {
+    open Microsoft.Quantum.Targeting;
+
+    /// # Summary
+    /// Applies the two qubit Ising $XX$ rotation gate.
```

Comment:
I see that in the code in multiple places (in other files) you replace the fragment `Ising` with fragment `R`, but in this line the fragment `Ising` survives. I just wanted to make sure that this is by design. Applicable to about 8 more files.

Reply:
That is intentional, as the name is still descriptive for the gates in text. For example, they are described on Wikipedia as:

> The qubit-qubit Ising coupling or Heisenberg interaction gates Rxx, Ryy and Rzz are 2-qubit gates that are implemented natively in some [trapped-ion quantum computers](https://en.wikipedia.org/wiki/Trapped_ion_quantum_computer), using for example the [Mølmer–Sørensen gate procedure](https://en.wikipedia.org/wiki/M%C3%B8lmer%E2%80%93S%C3%B8rensen_gate).

from [List of quantum logic gates](https://en.wikipedia.org/wiki/List_of_quantum_logic_gates#:~:text=The%20qubit-qubit%20Ising%20coupling%20or%20Heisenberg%20interaction%20gates,be%20expressed%20in%20sinusoidal%20form%20also%2C%20for%20example.).